sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -0,0 +1,593 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  WebResource,
+  ServiceClient,
+  ServiceClientOptions,
+  HttpOperationResponse,
+  AtomXmlOperationSpec,
+  signingPolicy,
+  userAgentPolicy,
+  logPolicy,
+  proxyPolicy,
+  atomSerializationPolicy,
+  RequestPolicyFactory,
+  AtomXmlSerializer,
+  URLBuilder,
+  ProxySettings
+} from "@azure/core-http";
+
+import { httpAtomXml } from "./log";
+import { SasServiceClientCredentials } from "./util/sasServiceClientCredentials";
+import * as Constants from "./util/constants";
+
+import { QueueResourceSerializer, QueueOptions } from "./serializers/queueResourceSerializer";
+import { TopicResourceSerializer, TopicOptions } from "./serializers/topicResourceSerializer";
+import {
+  SubscriptionResourceSerializer,
+  SubscriptionOptions
+} from "./serializers/subscriptionResourceSerializer";
+import { RuleResourceSerializer, RuleOptions } from "./serializers/ruleResourceSerializer";
+
+import {
+  ListRequestOptions,
+  QueueResult,
+  ListQueuesResult,
+  TopicResult,
+  ListTopicsResult,
+  SubscriptionResult,
+  ListSubscriptionsResult,
+  RuleResult,
+  ListRulesResult
+} from "./managementOperationInterfaces";
+
+/**
+ * All internal operations return a `Promise<HttpOperationResponse>` object of which the `parsedBody`
```

Comment:
This is no longer true right? Because we now return strongly typed request specific objects

Reply:
This still needs to be updated, you've done a lot of work to improve the response types :)